RepresentsMe/ViewControllers/Settings/BiometricSettingsViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  BiometricSettingsViewController.swift
+//  RepresentsMe
+//
+//  Created by Jacob Hausmann on 4/23/19.
+//  Copyright © 2019 11foot8. All rights reserved.
+//
+
+import UIKit
+
+class BiometricSettingsViewController: UIViewController {
+
+    override func viewDidLoad() {
```

Comment:
Is this class necessary?

Reply:
It will be, I'm working on adding faceID authentication